src/wp-includes/media.php

```diff
@@ -5001,5 +5002,90 @@ function wp_getimagesize( $filename, &$imageinfo = array() ) {
 	 *
 	 * phpcs:ignore WordPress.PHP.NoSilencedErrors
 	 */
-	return @getimagesize( $filename, $imageinfo );
+	return @_get_image_size( $filename, $imageinfo );
```

Comment:
```suggestion
	return @_wp_get_image_size( $filename, $imageinfo );
```

Reply:
Ah, this should be `_wp_get_image_size`; will fix.

Reply:
Fixed.